README.aspnet.md

```diff
@@ -0,0 +1,103 @@
+﻿# Featured Tags
+
+* `4.8`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:4.8`
+* `3.5`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:3.5`
+
+# About This Image
+
+ASP.NET is a high productivity framework for building Web Applications using Web Forms, MVC, Web API and SignalR.
+
+This repository contains `Dockerfile` definitions for ASP.NET Docker images. 
+
+This image contains:
+- Windows Server Core as the base OS
+- IIS 10 as Web Server
+- .NET Framework (multiple versions available)
+- .NET Extensibility for IIS
+
+# How to use these Images
+
+The [.NET Framework Docker samples](https://github.com/microsoft/dotnet-framework-docker/blob/master/samples/README.md) show various ways to use .NET Framework and Docker together.
+
+## Container sample: Run an ASP.NET application
+You can quickly run a container with a pre-built [sample ASP.NET Docker image](https://hub.docker.com/_/microsoft-dotnet-framework-samples/), based on the [ASP.NET Docker sample].
+
+Type the following [Docker](https://www.docker.com/products/docker) command:
+
+```console
+docker run --name aspnet_sample --rm -it -p 8000:80 mcr.microsoft.com/dotnet/framework/samples:aspnetapp
+```
+
+After the application starts, navigate to `http://localhost:8000` in your web browser. You need to navigate to the application via IP address instead of `localhost` for earlier Windows versions, which is demonstrated in [View the ASP.NET app in a running container on Windows](https://github.com/microsoft/dotnet-framework-docker/blob/master/samples/aspnetapp/README.md#view-the-aspnet-app-in-a-running-container-on-windows).
+
+For a comprehensive tutorial on running an ASP.NET app in a container, check out [the tutorial on the docs site](https://docs.microsoft.com/en-us/dotnet/articles/framework/docker/aspnetmvc).
+
+# Related Repos
+
+.NET Framework:
+
+* [microsoft/dotnet-framework](https://hub.docker.com/r/microsoft/dotnet-framework/): .NET Framework
+* [dotnet/framework/sdk](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/): .NET Framework SDK
+* [dotnet/framework/samples](https://hub.docker.com/_/microsoft-dotnet-framework-samples/): .NET Framework, ASP.NET and WCF Samples
+
+.NET Core:
+
+* [dotnet/core](https://hub.docker.com/_/microsoft-dotnet-core/): .NET Core
+* [microsoft/aspnetcore](https://hub.docker.com/r/microsoft/aspnetcore/) for ASP.NET Core applications.
+
+
+# Full Tag Listing
+
+## Windows Server, version 1903 amd64 Tags
+Tag | Dockerfile
+---------| ---------------
+4.8-20190709-windowsservercore-1903, 4.8-windowsservercore-1903, 4.8, latest | [Dockerfile](https://github.com/microsoft/dotnet-framework-docker/blob/master/4.8/runtime/windowsservercore-1903/Dockerfile)
```

Comment:
The Dockerfile links are not correct.  The [Get-TagsDocumentation script ](https://github.com/microsoft/dotnet-framework-docker/blob/master/eng/Get-TagsDocumentation.ps1)should be updated to work for ASP.NET and it should be run to update this section.  The script utilizes Linux images so you need to run it from a Linux container environment.